pkg/init/init

```diff
@@ -2,32 +2,11 @@
 
 setup_console() {
 	tty=${1%,*}
-	speed=${1#*,}
 	inittab="$2"
-	securetty="$3"
-	line=
-	term="linux"
-	[ "$speed" = "$1" ] && speed=115200
-
-	case "$tty" in
-	ttyS*|ttyAMA*|ttyUSB*|ttyMFD*)
-		line="-L"
-		term="vt100"
-		;;
-	tty?)
-		line=""
-		speed="38400"
-		term=""
-		;;
-	esac
 	# skip consoles already in inittab
 	grep -q "^$tty:" "$inittab" && return
 
 	echo "$tty::once:cat /etc/issue" >> "$inittab"
```

Comment:
This `/init` is only there to do a specific weird function that hopefully we can remove one day and then exec the real init. The less in it the better,

No, I meant add a `/etc/init.d/00/whale` that prints the whale on each console. This should then be run before `containerd` automatically.